Error: Cannot find module '../../razzle-dev-utils/logger'

## 🐛 Bug report
Incorrect import for logger in config/modules.js

```
Error: Cannot find module '../../razzle-dev-utils/logger'
Require stack:
- /my-app/node_modules/razzle/config/modules.js
- /my-app/node_modules/razzle/config/createConfig.js
- /my-app/node_modules/razzle/scripts/build.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/my-app/node_modules/razzle/config/modules.js:6:16)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/my-app/node_modules/razzle/config/modules.js',
    '/my-app/node_modules/razzle/config/createConfig.js',
    '/my-app/node_modules/razzle/scripts/build.js'
  ]
}
```

### Current Behavior

Any 3.1.x version does not build right now because of the incorrect import statement.

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Razzle           | 3.1.2
| Node             | 12.16.1
| npm         | 6.13.4
| Operating System | Mojave
| React            | 16.13.1

--- a/packages/razzle/config/modules.js
+++ b/packages/razzle/config/modules.js
@@ -3,7 +3,7 @@
 const fs = require('fs');
 const path = require('path');
 const paths = require('./paths');
-const logger = require('../../razzle-dev-utils/logger');
+const logger = require('razzle-dev-utils/logger');
 const resolve = require('resolve');
 const nodePath = require('./env').nodePath;
 

